modules/core/output_modules.php

```diff
@@ -177,28 +183,60 @@ class Hm_Output_login_start extends Hm_Output_Module {
      * Looks at the current login state and outputs the correct form
      */
     protected function output() {
+        $config = new Hm_Site_Config_File(CONFIG_FILE);
```

Comment:
Same as above - config is available in handler modules - pass whatever is needed to output modules and don't re-initialize the config class again here in an output module.